policy/datasets.yaml

```diff
@@ -1333,6 +1333,14 @@ analysisVisitTable_commonZp:
     storage: ParquetStorage
     python: lsst.pipe.tasks.parquetTable.ParquetTable
     template: plots/%(filter)s/tract-%(tract)d/visit-%(visit)d%(subdir)s/%(tract)d_%(visit)d_commonZp.parq
+analysisMatchRefVisitTable:
+    description: >
+        Per-visit table (for specific tract) of matched and denormalized
```

Comment:
I'm not sure what "denormalized" means.

Reply:
I appreciate your confusion as I had the same when I first encountered this term in the stack.  I inherited/adopted it since it is fairly widely used, e.g. https://github.com/search?q=org%3Alsst+denormalized&type=Code
(Note, in particular, `meas_astrom/python/lsst/meas/astrom/denormalizeMatches.py` & https://github.com/lsst/obs_base/blob/master/policy/datasets.yaml#L676-L684, which may actually suggest I should include`Full` in the name 😉).
The crux of it is that a “normalized” catalog only contains data IDs (and maybe a coord) to minimize space when persisting.  I’m explicitly referring to these as “demoralized” to indicate that they include the full set of catalog info for both `src` & `ref` cats.  If this adds more confusion than help, I’m happy to leave out that term (or use a more self-explanatory one if you have a suggestion!).

Reply:
Can you replace this with `denormalized (contains all columns from XX)`?

Reply:
Of course.  And looking at the precedent set above, I am leaning strongly towards adding "Full" to the dataset names, i.e 
```
analysisMatchFullRefVisitTable
analysisMatchFullRefCoaddTable_forced
analysisMatchFullRefCoaddTable_unforced
```
Do you agree?